tests/libcxx/skipped_tests.txt

```diff
@@ -498,24 +498,11 @@ thread\futures\futures.task\futures.task.members\make_ready_at_thread_exit.pass.
 
 
 # *** C1XX COMPILER BUGS ***
-# Compiler bug: VSO-120957 "alignas by-value parameters should be permitted"
-utilities\meta\meta.trans\meta.trans.other\aligned_storage.pass.cpp

```

Comment:
That was one of the outstanding questions I had. I would have to teach our internal msvc executor how to deal with the two files (and also the `\\{1,0}` test names).

Reply:
We'll have to do _something_; we literally can't check this in as-is since it will break the PR build and the internal branch policies require a passing build. We had vaguely discussed changing the internal test executor to use this machinery directly (eventually) but that's more work and I'd be uncomfortable doing so anyway without us banging on it for a month or so first to find any problems.

Should we back out this part? Maybe use different a skip list file (or only use `expected_results`?) for this executor and double our skipped-test maintenance load until we unify the internal/external executor machinery?

Reply:
Sorry I was being unclear. When I said "Is this worth the effort" I was referring to the increased effort of needing to modify our internal test runner with the implication that I would back out this change otherwise and continue skipping all the tests.

I prefer your suggestion of backing out the changes to `skipped_tests.txt` and moving all the logic into `expected_results.txt` I think the dual effort of modifying both files would be minimal.

Reply:
> I think the dual effort of modifying both files would be minimal as they would be so similar.

It would certainly be less troublesome than our current dual-PR process, and I think it's clearly on the path from where we are now to where we eventually want to be (using just list of tests, and using this test machinery internally as well as externally). 

Do I conjecture correctly that we can add entries into `expected_results` with a `SKIP` tag and avoid running them at all? Some of our skips are "we really don't want to run this at all" rather than "this won't run", e.g., there's an iostreams test that takes hours to run.

Reply:
The machinery is currently not in place to allow the `expected_results` file to skip a test but it would be trivial to add it, which is what I'm planning on doing to resolve this issue.